pkg/operator/starter.go

```diff
@@ -33,12 +32,18 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		return err
 	}
 
-	genericOperatorClient, dynamicInformers, err := genericoperatorclient.NewClusterScopedOperatorClient(
-		cc.KubeConfig, operatorv1.GroupVersion.WithResource("certmanagers"))
+	certManagerOperatorClient, err := operatorclient.NewForConfig(cc.KubeConfig)
 	if err != nil {
 		return err
 	}
 
+	certManagerInformers := operatorinformers.NewSharedInformerFactory(certManagerOperatorClient, 10*time.Second)
```

Comment:
resync every 10 seconds seems short, maybe @sttts or @p0lyn0mial might suggest a better value

Reply:
thank you for the sharp eyes, I actually wanted to set `10*time.Minute`. the dynamic client sets it to 12h (huh) [1] which seems quite large, I'll set to 10 minutes as a first guess.

[1] https://github.com/openshift/library-go/blob/7aab941b4a2b60dc5687744944789456d7a4eb4e/pkg/operator/genericoperatorclient/dynamic_operator_client.go#L33